PlayFabSDK/source/Qos/QosResult.cs

```diff
@@ -5,36 +5,35 @@ namespace PlayFab.QoS
 
     public class QosResult
     {
-        public List<QosRegionResult> RegionResults;
+        public List<QosRegionResult> RegionResults { get; set; }
```

Comment:
For client performance reasons, we use fields rather than properties in the SDKs.